### ISSUE TITLE
Option to suppress warning/error messages that aren't prefixed with [...] without suppressing everything else

### ISSUE THREAD

**Issue by bpsib** [mpv-player/mpv#11203]
it's great that mpv has an option to suppress certain messages using the `--msg-level=` option but there are many warning messages that aren't prefixed so can't suppress without suppressing everything else. An example is the annoying Audio/Video desynchronisation detected warning message. I don't notice any desynchronisation so is not needed for me. One of the others is the audio device underrun message.

### Expected behavior of the wanted feature
The ability to suppress terminal message using wildcard matches. E.g. `--msg-level="*Audio/Video desynchronisation*=no"`

**Comment by Dudemanguy** [comment]
You should be able to control those though. They should be under `cplayer`.

**Comment by bpsib** [comment]
> You should be able to control those though. They should be under `cplayer`.

The problem is with that is it also hides useful info messages. It is possible to hide `cplayer` warning messages only?

**Comment by Dudemanguy** [comment]
Yeah there's not currently a way to pick a msg-level at specific log levels. Suppression works via hiding messages at that certain log level and greater (i.e. if you hide warnings than everything more noisy is hidden too).

**Comment by bpsib** [comment]
It's a shame that no one cares to do anything about this. The error message DRIVES ME BLOODY INSANE

**Comment by NRKWL** [comment]
I use [auto-save-state.lua](https://github.com/dyphire/mpv-config/blob/master/scripts/auto-save-state.lua). It runs every 60 seconds. I am spammed with "Saving state". I'd like to filter logs by regex.

**Comment by gevvan** [comment]
I jump in here á la classic thread necromancy.
Nah, jokes put aside, I suffer from this, too.

client removed during hook handling
↑
This message comes up in yellow every now and then when launching my newly installed v0.4.0.
With older versions I never had it. I don't care where it comes from, I don't want to see it.
How can I suppress it without ignoring everything else? Or am I all out of luck?

**Comment by RundownRhino** [comment]
This works to locally suppress "Saving state." info messages when saving state in scripts. This message has no prefix so we have to target all `cplayer` info messages:
```lua
local old = mp.get_property("msg-level")
mp.set_property("msg-level", old == "" and "cplayer=warn" or old .. ",cplayer=warn")
mp.command("write-watch-later-config") -- normally says "Saving state."
mp.set_property("msg-level", old)
```
